api/src/test/java/org/apache/commons/rdf/api/RDFSyntaxTest.java

```diff
@@ -114,8 +116,7 @@ public void name() throws Exception {
 
     @Test
     public void valueOf() throws Exception {
-        assertEquals(RDFSyntax.TURTLE, RDFSyntax.valueOf("TURTLE"));
-        // No need to test all of them, we'll trust Enum
+        assertEquals(RDFSyntax.TURTLE, RDFSyntax.byName("TURTLE"));
```

Comment:
`RDFSyntax::byName` returns an `Optional`. Perhaps you mean:

    assertEquals(RDFSyntax.TURTLE, RDFSyntax.byName("TURTLE").get());